tests/integration/aiplatform/test_dataset.py

```diff
@@ -0,0 +1,278 @@
+        """Retrieve new, empty dataset and import a text dataset using import().
+        Then verify data items were successfully imported."""
+
+        assert shared_state["dataset_name"]
+        aiplatform.init(project=_TEST_PROJECT, location=_TEST_LOCATION)
+
+        my_dataset = aiplatform.Dataset(dataset_name=shared_state["dataset_name"])
+
+        data_items_pre_import = dataset_gapic_client.list_data_items(
+            parent=my_dataset.resource_name
+        )
+
+        assert len(list(data_items_pre_import)) == 0
+
+        # Blocking call to import
+        my_dataset.import_data(
+            gcs_source=_TEST_TEXT_ENTITY_EXTRACTION_GCS_SOURCE,
+            import_schema_uri=_TEST_TEXT_ENTITY_IMPORT_SCHEMA,
+        )
+
+        data_items_post_import = dataset_gapic_client.list_data_items(
+            parent=my_dataset.resource_name
+        )
+
+        assert len(list(data_items_post_import)) > 400  # Expected value 469
+
+    @pytest.mark.usefixtures("delete_new_dataset")
+    def test_create_and_import_image_dataset(self, dataset_gapic_client, shared_state):
+        """Use the Dataset.create() method to create a new image obj detection
+        dataset and import images. Then confirm images were successfully imported."""
+
+        aiplatform.init(project=_TEST_PROJECT, location=_TEST_LOCATION)
+
+        img_dataset = aiplatform.Dataset.create(
+            display_name=f"temp_sdk_integration_create_and_import_dataset_{uuid.uuid4()}",
+            metadata_schema_uri=aiplatform.schema.dataset.metadata.image,
+            gcs_source=_TEST_IMAGE_OBJECT_DETECTION_GCS_SOURCE,
+            import_schema_uri=_TEST_IMAGE_OBJ_DET_IMPORT_SCHEMA,
+        )
+
+        data_items_iterator = dataset_gapic_client.list_data_items(
+            parent=img_dataset.resource_name
+        )
+
+        assert len(list(data_items_iterator)) > 10  # Expected value 14
+
+        shared_state["dataset_name"] = img_dataset.resource_name
+
+    @pytest.mark.usefixtures("delete_new_dataset")
+    def test_create_tabular_dataset(self, dataset_gapic_client, shared_state):
+        """Use the Dataset.create() method to create a new tabular dataset.
+        Then confirm the dataset was successfully created and references GCS source."""
+
+        aiplatform.init(project=_TEST_PROJECT, location=_TEST_LOCATION)
+
+        tabular_dataset = aiplatform.Dataset.create(
+            display_name=f"temp_sdk_integration_create_and_import_dataset_{uuid.uuid4()}",
+            metadata_schema_uri=aiplatform.schema.dataset.metadata.tabular,
+            gcs_source=[_TEST_TABULAR_CLASSIFICATION_GCS_SOURCE],
+        )
+
+        gapic_dataset = tabular_dataset._gca_resource
+
+        gapic_metadata = json_format.MessageToDict(gapic_dataset._pb.metadata)
+        gcs_source_uris = gapic_metadata["inputConfig"]["gcsSource"]["uri"]
+
+        assert _TEST_TABULAR_CLASSIFICATION_GCS_SOURCE in gcs_source_uris
```

Comment:
Can this test be more specific and test that the resource has exactly this one uri?

Reply:
Yes, I'll change it to do so